libs/langchain/langchain/chat_models/huggingface.py

```diff
@@ -44,14 +41,20 @@ class ChatHuggingFace(BaseChatModel):
 
     llm: LLM
     system_message: SystemMessage = SystemMessage(content=DEFAULT_SYSTEM_PROMPT)
-    tokenizer: AutoTokenizer = None
-    model_id: str = None # type: ignore
+    tokenizer: None
+    model_id: str = None  # type: ignore
 
     def __init__(self, **kwargs: Any):
         super().__init__(**kwargs)
 
-        if not isinstance(self.llm, (HuggingFaceEndpoint, HuggingFaceTextGenInference, HuggingFaceHub)):
-            raise ValueError(f"LLM type not supported for ChatHuggingFace: {type(self.llm)}")
+        from transformers import AutoTokenizer
```

Comment:
@A-Roucher This needs to be a local import per [this comment](https://github.com/langchain-ai/langchain/pull/14040#discussion_r1411042457) by @baskaryan 

With this change, it seems we're back to failing a few tests again :(